### ISSUE TITLE
'nucypher stake events' reports invalid ethereum address for valid addresses.

### ISSUE THREAD

**Issue by KPrasch** [nucypher/nucypher#2559]
**Describe the Bug**
A clear and concise description of what the bug is, and what you expected to happen.

**To Reproduce**
```
nucypher stake events --console-logs --staking-address 0xb53507dfD915dBD880676Cf703Ff18C62cD5a0BC --provider https://mainnet.infura.io/v3/... --event-name Withdrawn
```

**Traceback or Screenshots (Optional)**

![Screenshot from 2021-02-08 15-14-27](https://user-images.githubusercontent.com/679404/107293385-602d2080-6a20-11eb-85a5-3a23f534fec6.png)

**System (please complete the following information):**
 - OS Version: Ubuntu 20.04
 - Nucypher Version: 4.6.0
 - Ethereum Node Version: Infura

**Comment by derekpierre** [comment]
(@KPrasch be careful with your infura endpoint URI which is displayed in the screenshot).

This is what I get when I run the command

```
nucypher.blockchain.eth.actors.UnknownAccount: 'Account 0xb53507dfD915dBD880676Cf703Ff18C62cD5a0BC is not known by this Ethereum client. Is it a HW account? If so, make sure that your device is plugged in and you use the --hw-wallet flag.'
```

**Comment by KPrasch** [comment]
Closed by #2867
